mail_autosubscribe/tests/models/fake_order.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Camptocamp (http://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class FakeOrder(models.Model):
```

Comment:
res.partner already inherits from mail.thread, you could maybe work w/ it for tests?
Not a blocker.

Reply:
I thought so but I figured this makes for a more real-case use case